src/coreclr/tools/aot/ILCompiler.RyuJit/JitInterface/CorInfoImpl.Llvm.cs

```diff
@@ -28,6 +28,7 @@ public static void addCodeReloc(IntPtr thisHandle, void* handle)
                         if (_this._compilation.HasLazyStaticConstructor(target))
                         {
                             var nonGcStaticSymbol = _this._compilation.NodeFactory.TypeNonGCStaticsSymbol(target);
+                            _this._codeRelocs.Add(new Relocation(RelocType.IMAGE_REL_BASED_REL32, 0, nonGcStaticSymbol));
```

Comment:
I'm revisiting https://github.com/dotnet/runtimelab/pull/890#issuecomment-813105021 as although the GCStatics symbol was added, the NonGCStatics symbol was added as a symbol, but never made it to a dependency so was never initialised.